text/0000-cfg-path-version.md

```diff
@@ -508,22 +508,27 @@ with `GHC` and alternative Haskell compilers.
 
 ### The bikeshed - Argument syntax
 
-We have two options with respect to how the `version` flag may be specified:
+We have roughly two options with respect to how the `version` flag may be specified:
 
 1. `version = "<semver>"`
-2. `version("<semver>")`
+2. `version(<semver>)`
 
 The syntax in 2. is currently an error in `#[cfg(..)]` as you may witness with:
 
 ```rust
 // error[E0565]: unsupported literal
-#[cfg(abracadabra("1.27"))] fn bar() {}
+#[cfg(abracadabra(1.27))] fn bar() {}
+                  ^^^^
 ```
 
-We could allow this syntax. However, we have chosen the syntax in 1.
-This with consistency with flags such as `target_feature = "bmi"`.
-Another reason to go with `version = ".."` is that `version("..")`
-looks like a list.
+However, we have decided to go with the syntax in 2. because the `cfg` flags
```

Comment:
Should mention that arbitrary tokens are technically allowed inside the `...` part of `#[attr(...)]`, so extending the grammar *within* a builtin attribute should be possible.